behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx

```diff
@@ -179,7 +181,7 @@ export const MessagesPage = () => {
             routeToChat(0);
         }
         // close the modal
-        onCloseModal();
+        //onCloseModal();
```

Comment:
This might be causing the reloop not sure

Reply:
I found the fix for it!